ruby_exercises/mythical-creatures/lib/medusa.rb

```diff
@@ -0,0 +1,26 @@
+class Medusa
+  attr_reader :name,
+              :statues
+
+  def initialize(name)
+    @name = name 
+    @statues = []
+  end
+
+  def stare(victim)
+    @statues.push(victim)
+    victim.stoned = true 
+  end
+  
+  def three_victims?
+    @statues.count == 3 
+  end
+
+  def fourth_victim
+    if @statues.count > 3 
+      @statues.first.stoned = false 
+    else
+      #Not sure what to put here or if I even need an else. Can I just have an if?
```

Comment:
You don't need an else, you can have if statements on their own as branches of logic. I would say that it's a bit of a code smell to have a method contain just an if and no other logic after it -- I would take it to mean that the logic from `three_victims?` and `fourth_victim` can be combined into a single method, especially if they're both returning a boolean value.